src/App.tsx

```diff
@@ -1,10 +1,41 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import { useEffect, FC, useState, useMemo } from 'react';
+import { getTodos } from './api/todos';
 import { UserWarning } from './UserWarning';
+import { TodoList } from './Components/TodoList';
 import { USER_ID } from './api/todos';
+import cn from 'classnames';
+import { Todo } from './types/Todo';
+import { Header } from './Components/Header';
+import { Filters } from './Components/Filters';
+import { TodoFilter } from './Components/TodoFilter';
+
+export const App: FC = () => {
+  const [currentFilter, setCurrentFilter] = useState(Filters.All);
+  const [tasks, setTasks] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    getTodos()
```

Comment:
useEffect(() => {
  setErrorMessage(''); 
  getTodos()
    .then(setTasks) 
    .catch(() => {
      setErrorMessage('Unable to load todos'); 
      setTimeout(() => setErrorMessage(''), 3000); 
    });
}, []);

To ensure correct handling of error messages, you must use setErrorMessage (") before calling getTodos ()